source/frontend/src/features/projects/detail/projectPages/details/ProjectDetailInformation.tsx

```diff
@@ -0,0 +1,17 @@
+import { Section } from 'features/mapSideBar/tabs/Section';
+import { getIn, useFormikContext } from 'formik';
+import { IProjectForm } from 'interfaces/IProject';
+import { withNameSpace } from 'utils/formUtils';
+
+export interface IProjectDetailInformationProps {
+  nameSpace?: string;
+}
+
+export const ProjectDetailInformation: React.FunctionComponent<
+  React.PropsWithChildren<IProjectDetailInformationProps>
+> = ({ nameSpace }) => {
+  const { values } = useFormikContext<IProjectForm>();
+  const summary = getIn(values, withNameSpace(nameSpace, 'note'));
+
+  return <Section>This works!{summary}</Section>;
```

Comment:
Are we sure we want to display "This works!" ?

Reply:
Removed